personalized_nlp/datasets/emotions_perspective/emotions_perspectives.py

```diff
@@ -42,6 +44,7 @@ def __init__(
         self.classification = classification
         self.normalize = normalize
         self.min_annotations_per_text = min_annotations_per_text
+        self.regression = False
```

Comment:
`self.regression` should be set based on the argument from the constructor. Of course its value by default can be `False`. 

P.S. Hmm. there is even an argument `classification` which is assigned but never used in the code.